### DISCUSSION TITLE
volumetric properties

### DISCUSSION THREAD

**Discussion by SJT77** [usnistgov/REFPROP-issues#660] [Ideas]
I have an interest in making some plots of some thermodynamic functions normalized to volume instead of mass/moles. I was wondering if there is a version of REFPROP that allows for output to be generated / plotted on a volumetric basis?

**Comment by ianhbell** [comment]
In short, not really. The notion of "per volume" is not well defined for mixtures because while mass and/or moles are additive, volume is not, and as a consequence, even the question of "per volume" depends on a model for the excess volume, which requires a thermodynamic model of some kind, even a simple one.

**Reply by SJT77** [reply]
Ok. Thanks, I can scrape through what I need to do using only available features.
